### ISSUE TITLE
Nr 89 wee heavy scotch

### ISSUE THREAD

**Issue by Bbbolaget-karl** [sweco-seukni/Bbbolaget#85]

**Comment by Bbbolaget-karl** [comment]
http://www.homebrewtalk.com/showthread.php?t=163985

En tråd om BU och förhållandet till IBU. 

Hittar recept om 
Pale ale
Carapils
Rostat korn
Lite oklara procent. 

Tänker att vi kör 4 kg malt i påsen. Dekoktionskokar en del och tillsätter strösocker om dt behövs. Helst ska vi få med lite strösocker, så får vi späda också

**Comment by Bbbolaget-karl** [comment]
Om vi vill ha invertsocker att ha i.
Recept plockat från forum

1,35kg råsocker
1,5L vatten
1gr askorbinsyra/kg socker
120ml Glykos

Koka upp vatten och lös sockret tillsammans med askorbinsyran och glykosen. Låt koka tills du kommer upp i 110°. Nu får du sänka värmen på plattan för annars går det för fort till 114°. Efter 20 minuter vid 114° har du nu Invert socker nr1. Låt sjuda vidare vid 114° till det har mörknat till ungefär mörk sirap, ca 1timme. Nu har du invert socker nr2, eller så låter du det bli mer eller mindre svart vid 114° då får du invert socker nr3 men då tar det 2-3 timmar.

Glykosen har man i för att det inte ska kristallisera sig, askorbinsyran kan bytas ut mot Vinsten, citronsyra eller mjölksyra om man vill. Det hjälper till vid spjälkningen enl Wikipedia.
När man väl kommit upp till 114° är det svårt att få tillräckligt låg värme på plattan, det var faktiskt det största problemet. Helst ska tempen inte upp alls men det var ok om man kom upp till 118° men gick det över det fick man brända smaker som inte var önskvärda

**Comment by Bbbolaget-karl** [comment]
http://qldkev.net/invertsugar.php
Ett annat recept på invertsocker. Verkar bättre utifrån vad man har hemma

**Comment by Bbbolaget-karl** [comment]
Förkultur på en påse Wyeast 1728, Scottisch ale. med magnetomrörare. Tis-tors. vila fredag. 
Jästen tål att jäsa så lågt som 13 gr. 
Tänker jäsa vid 14 gr de första 5 dagarna, sen lite oklart, men antagligen 20 gr ytterigare 10 dagar eller så.

**Comment by Bbbolaget-karl** [comment]
Hittat receptet på Traquair Scotch ale, eller i vart fall ngt från han som bryggern, fr boken The Secrets of Master Brewers. 
as follows (5 gallons, 19 liter):
12.25 pounds Brittisk pale alemalt (98%)
0,25 pound Roasted Barley (2%)
Mäska 64 gr i en timma
koka 120 min
26 IBU EKG at 120 min
3 IBU EKG at 10 min

Jäs med 1728, svalt
OG est 1070
Fg est 1015
avb est 7,2%
29 IBU

**Comment by Bbbolaget-karl** [comment]
Tweakar enligt följande
Malt
4 kg totalt
98% Maris Otter
2 % Roasted Barley (kolla upp så det finns hemma)

Mäska  ca 65-66 gr i en timma. (mäskvattnet till 71,0 gr C) Eg en approx; 3 kg och 70,5 gr ger 67 gr. Kan justeras

kokas i en timma
26 IBU EKG vid 60
3 IBU EKG vid 10
Vid kokstart justeras förkoksOG vid behov. förkoksOG est 1060. Innan eventuell spädning sker tas ca 3 (?) liter ut för inkokning vid separat kastrull, för ökad malliard och försockring. 
Den återföres strax innan 10 min innan kokslut, innan sista EKG går i, koksOG kollas, humlegivan justeras ev. 
Om behov av högre OG finns, i med lagom mängd Rårörsocker. (egentligen skulle jag spjälkat rårörsocker till invertsocker, men har inte hunnit med det, får bli till nästa gång.)

kyl och jäs enligt ovan.

**Comment by Bbbolaget-karl** [comment]
har kommit en ny engelsk maltsort på humleshoppen som verkar intressant. Får bli att köpa hem den till nån bryggning snart.

**Comment by sweco-seukni** [comment]
Blir knepigt att justera humlegivan i slutet av koket, jag tycker vi ska höja den första humlegivan, säg till 36 i IBU. Läropengen sist var att det blev för sött med sidokokningen, vill inte hamna där igen.

**Comment by Bbbolaget-karl** [comment]
jo, det blir lite knepigt med humlegivan. Men tror det kan vara värt
Såg nu kommentarerna från förra. Vi bör höja som du skriver. Det hänger väl på FG, men om det blir lågt FG och  därmed högt IBU får vi väl lagra den länge...

**Comment by Bbbolaget-karl** [comment]
den här har vi nog behövt länge...
http://nomograph.babbrewers.com/

**Comment by Bbbolaget-karl** [comment]
2 ml lactic acid

**Comment by Bbbolaget-karl** [comment]
80ng roasted barley från Crisp. Resten Maris otter. mals nu

**Comment by Bbbolaget-karl** [comment]
mäskvattnet blir 71 gr istf normala 70,5. Lite lägre mäskgrad och lite komepnsation, alltså

**Comment by Bbbolaget-karl** [comment]
EKG kottar bbf juni 2019
6,5 % alfasyra. 
33 IBU vid 60 min   27 gram
3 IBU vid 10 min      8 gram
förutsatt 16 liter kokvolym och 12 liter färdig volym (dekoktion) med ett förkoksG på 1053. Kan behöva justeras.

**Comment by Bbbolaget-karl** [comment]
OG 1066,5 efter mäskning

**Comment by Bbbolaget-karl** [comment]
14, 5 liter. 
1066
Vi gissar att vi vill ha FörkoksOG till 1058. Då späder vi med 2,0 liter.

**Comment by Bbbolaget-karl** [comment]
satsar på att vi kokar bort 3,0 liter. Då hamnar vi spot on enl beräkning, eller nästan 1071 i vart fall.

**Comment by Bbbolaget-karl** [comment]
EKG kottar bbf juni 2019
6,5 % alfasyra.
33 IBU vid 60 min 32 gram
3 IBU vid 10 min 8 gram
förutsatt 16,5 liter kokvolym och 13,5 liter färdig volym (dekoktion) med ett förkoksG på 1058. Är justerat.

**Comment by Bbbolaget-karl** [comment]
Efter kok
1076
13 liter

**Comment by Bbbolaget-karl** [comment]
Körde  nya luftsättaren, akvariepumpen, i 40 min. 30 min sägs ge de 8 ppm som eftersträvades. 
Lite funderingar kring rengöring av den där produkten. 

Glömde vi späda vörten? Skulle ju bli 1070. Inget minne av att vi spädde den. Gjorde vi?
i så fall ska det i en liter till 1070. 

Det jäser vid 14 gr i kyl, jäsning påbörjat såhär morgonen efter.

**Comment by Bbbolaget-karl** [comment]
konstaterat att vi glömde späda. Spädde nu istället. 1,0 liter. 
OG 1070. 
14 liter jäsvolym

**Comment by Bbbolaget-karl** [comment]
tisdag morgon ( bryggde fredag kväll). Kylen stängdes av, kylskåpsdörren öppen. bubblar ca var 3-4 sekund. 14 gr ca i kylskåp, ca 16 i källare. damejannen ska sen upp till rumstemp, imorgon morgon? Återstår att se.

**Comment by Bbbolaget-karl** [comment]
Test

**Comment by Bbbolaget-karl** [comment]
Fg 1016! Där satt den!

**Comment by Bbbolaget-karl** [comment]
Bara aningen stark, i nån nyans i smaken. Annars mycket fin! Sätts i kylskåp idag, 21 dec, för att låta kallagra sig ett par veckor.

**Comment by Bbbolaget-karl** [comment]
7,1 % abv

**Comment by Bbbolaget-karl** [comment]
Testat mot traquair jacobite ale. Vår är bättre. Receptet stämmer inte, det kommersiella, eller så är det en helt annan basmalt. 
De har nån muggig mellanto. Inbakad, carared? Inget bra
